.github/workflows/ci.yml

```diff
@@ -30,6 +30,8 @@ jobs:
         with:
           version: v1.56.2 # this is the golangci-lint version
           args: --timeout 5m0s
+          skip-build-cache: true
+          skip-pkg-cache: true
```

Comment:
was running into an issue w/ the linter in GH actions accessing existing go packages https://github.com/golangci/golangci-lint-action/issues/244#issuecomment-957713816

Reply:
👍  Oh, I've never seen that. This must have started after I bumped the version. Thanks for addressing it.